### PR TITLE
Rework floor health scanners + health status screens to be more useful

### PR DESCRIPTION
**Pull request by flrsh**
[REWORK] [CLEANLINESS] [ENHANCEMENT] 

## About the PR 

This PR makes floor health scanners and health status screens more useful so that more people will actually use them.

- Floor health scanners now start with reagent + organ scan capabilities.
- Walking on top of a floor health scanner will trigger a health scan and display the results to the user.
- If the user is in crit, a PDA alert will be sent to the medical department.
- Health status screens now have an increased examine range.

I also cleaned up some code but that doesn't really affect any functionality.

## Why's this needed? 

It's nice when things are actually used.

## Changelog

```
(u)Flourish:
(*)Made the floor health scanners in medbay more useful. Step on top of one of them to receive a readout of your health and trigger a PDA alert to medical if you're in crit. 
```

### CODE REVIEW

File: code/obj/health_scanner.dm [hunk 1]
```diff
@@ -27,84 +24,34 @@
...
-				myPartner.reagent_scan = 1
-			else if (myPartner.reagent_scan && !src.reagent_scan)
-				src.reagent_scan = 1
-
 /obj/health_scanner/wall
 	name = "health status screen"
 	desc = "A screen that shows health information recieved from connected floor scanners."
 	icon_state = "wallscan1"
+	var/list/partners // who do we know?
+	var/examine_range = 7 // from how far away can people examine the screen
```
Comment (ZeWaka) [inline code/obj/health_scanner.dm:38]: Is this supposed to be a hardcoded 7, or half of a screen height?

Comment (flrsh) [reply code/obj/health_scanner.dm:38]: I just put an arbitrary number but it'd be better if it depended on screen height. Should I just do something like `src.examine_range = (SQUARE_TILE_WIDTH - 1) / 2` under `New()`?

Comment (ZeWaka) [reply code/obj/health_scanner.dm:38]: i mean I'd just do it there, it's a define, but yes

Comment (flrsh) [reply code/obj/health_scanner.dm:38]: That's neat, ty.

**Pull request merged**